tests/test_inotify_c.py

```diff
@@ -60,20 +74,18 @@ def test_late_double_deletion(monkeypatch):
     # CREATE DELETE CREATE DELETE DELETE_SELF IGNORE DELETE_SELF IGNORE
     inotify_fd.buf = (
         # IN_CREATE|IS_DIR (wd = 1, path = subdir1)
-        b"\x01\x00\x00\x00\x00\x01\x00\x40\x00\x00\x00\x00\x10\x00\x00\x00"
-        b"\x73\x75\x62\x64\x69\x72\x31\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+        struct_inotify(1, 0x40000100, 0, 16, b"subdir1") +
         # IN_DELETE|IS_DIR (wd = 1, path = subdir1)
-        b"\x01\x00\x00\x00\x00\x02\x00\x40\x00\x00\x00\x00\x10\x00\x00\x00"
-        b"\x73\x75\x62\x64\x69\x72\x31\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+        struct_inotify(1, 0x40000200, 0, 16, b"subdir1")
     ) * 2 + (
         # IN_DELETE_SELF (wd = 2)
-        b"\x02\x00\x00\x00\x00\x04\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+        struct_inotify(2, 0x00000400, 0, 0, b"") +
```

Comment:
Maybe could we use `InotifyConstants` here to ease reading? https://github.com/gorakhargosh/watchdog/blob/14152385f684fccb59f03c39a64edd7d89241d2a/src/watchdog/observers/inotify_c.py#L45

Reply:
Sure, let's try.

Reply:
Done. I've also switched to named parameters and added some zero defaults to shorten the code a bit.

Reply:
It is so much better, thanks a lot!